rust/README.md

```diff
@@ -0,0 +1,42 @@
+# Rust on CodeQL
+
+## Development
+
+### Dependencies
+
+At present you need to be in the `rust-experiment` branch (or a branch on top of that) in both the `semmle-code` and `codeql` repos.
+
+TODO: are there any dependencies that aren't already CodeQL dependencies?  bazel and rustc?
```

Comment:
for bazel if one has the internal repo they can use `tools/bazel`. Otherwise [bazelisk](https://github.com/bazelbuild/bazelisk) can be used as well. When building with this command no dependency on rust/cargo is there (everything is provided for by `bazel`)

Reply:
I've added the link to Bazelisk.

Reply:
I'm never quite sure which bits of dev env setup are required vs optional, but we'll go with it.  Thanks.